src/Enzyme.jl

```diff
@@ -394,7 +402,7 @@ f(x) = x*x
 """
 @inline function autodiff(::ForwardMode, f::F, ::Type{A}, args...) where {F, A<:Annotation}
     args′  = annotate(args...)
-    if any_active(args′...)
+    if any_active(args...)
```

Comment:
Why this change?

Reply:
any_active wasn't getting optimized out (which presumably was fixed by the tuple defn rather than recurrence, rather than this).